apps/mobile/src/screens/tasks/tasks.tsx

```diff
@@ -36,6 +38,15 @@ const Tasks: React.FC = () => {
 		TasksMode.CURRENT,
 	);
 
+	const currentTasks: TaskDto[] = tasks.filter(
+		(task) => task.status === TaskStatus.CURRENT,
+	);
+	const pastTasks: TaskDto[] = tasks.filter(
+		(task) => task.status === TaskStatus.COMPLETED,
+	);
+	const isCurrentEmpty = currentTasks.length === NumericalValue.ZERO;
+	const isPastEmpty = pastTasks.length === NumericalValue.ZERO;
+
```

Comment:
Why do we need this? Regarding the logic below - we already have filtered tasks returned from the store.